src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -67,13 +68,13 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         try {
             $response = $event->getKernel()->handle($request, HttpKernelInterface::SUB_REQUEST, true);
         } catch (\Exception $e) {
-            $this->logException($exception, sprintf('Exception thrown when handling an exception (%s: %s)', get_class($e), $e->getMessage()), false);
+            $this->logException($e, sprintf('Exception thrown when handling an exception (%s: %s at %s line %s)', get_class($e), $e->getMessage(), $e->getFile(), $e->getLine()), false);
```

Comment:
The context is already logged, see https://github.com/symfony/symfony/pull/13767/files#diff-3a012244a1c6d5c901c4f26d6b2ca299R102